source/upgrade/upgrade_notes.rst

```diff
@@ -12,6 +12,8 @@ Upgrade notes
   not backup or synchronized anymore.
 * The `/var/lib/xivo/sounds` directory has been migrated to `/var/lib/wazo/sounds` and the directory
   `/var/lib/xivo` is considered deprecated. Please update all custom references to this path.
+* `xivo-provisioning` API URL has been updated to remove the `provd` prefix and add the API version
+  number, which is `0.2`. All affected services and `wazo-provd-client` have been updated.
```

Comment:
Could you provide a "before" and "after" URL as an example?

Reply:
Fixed